spatial.js

```diff
@@ -0,0 +1,27 @@
+if (typeof (Number.prototype.toRad) === 'undefined') {
+  Number.prototype.toRad = function () {
+    return this * Math.PI / 180;
+  };
+}
+
+module.exports = {
+  distance: distance
+};
+
+function distance(start, end) {
```

Comment:
IMO you should use a external npm gps util like 

https://github.com/vanng822/gps-util#getdistancelng1-lat1-lng2-lat2